.eslintrc.js

```diff
@@ -8,11 +8,11 @@ module.exports = {
     'plugin:@typescript-eslint/recommended',
     'plugin:vue/base',
     'plugin:vue/essential',
-    // 'plugin:vue/recommended',
-    // 'plugin:vue/strongly-recommended',
+    'plugin:vue/recommended',
```

Comment:
This preset caused a lot of auto-fixing -- sorry -- but I think it's useful to have as it helps with templating styling.

The other 3 "recommended" presets below did nothing incremental (but still good to have).